eventPage.js

```diff
@@ -10,7 +10,7 @@ function startApplication() {
         id: 'main-window',
         frame: 'chrome',
         innerBounds: {
-            minWidth: 960,
+            minWidth: 1020,
```

Comment:
Is there a good reason for this?  Does the app still render correctly on small chromebooks for example?